view/vw_oo_overflow.yaml

```diff
@@ -3,10 +3,9 @@ table: qgep_od.overflow
 view_name: vw_qgep_overflow
 
 additional_columns:
-  geometry: ST_MakeLine(n1.situation_geometry, n2.situation_geometry)::geometry('LineString',%(SRID)s)
+  geometry: n1.situation_geometry::geometry('PointZ',%(SRID)s)
```

Comment:
This is now a point instead of a line - I am not sure if that is what @urskaufmann intended this to be.

Reply:
@cymed can you check with @urskaufmann if this still is what both of you expect of this?

Reply:
@sjib: @urskaufmann and I agreed to changing the geometry from a line to a point in November. I can forward you the discussions by mail.

Reply:
OK, then everything is ok